code/modules/mob/language/station.dm

```diff
@@ -97,4 +98,3 @@
 					 "portal", "systema", "electronika", "nigilizm", "anarhizm", "communistu", "tupuye", "sopla", "obshivka", "obtekaemost", \
```

Comment:
A lot of these syllables don't seem like syllables at all, but whole words

They're supposed to be little bits of words, pieced together to make a whole word.  Usually 2-4 letters
Six letters is kind of pushing it for a syllable, yet some of these have 11

i think they're too long, and so are the words that result from them

Reply:
I think it's intended to be just words. If the spaces inbetween them would be 100% chance it would be fine. (Most are Russian words in transleet).